src/auth/authRouter.ts

```diff
@@ -68,11 +68,7 @@ authRouter.post(`/logout`, async (req, res) => {
 });
 
 authRouter.post(`/check`, async (req, res) => {
-  if (req.session?.user) {
-    return res.json(req.session.user);
-  } else {
-    return res.sendStatus(401);
-  }
+  req.session?.user ? res.json(req.session.user) : res.sendStatus(401)
```

Comment:
Hi Matt. Chained ternaries make the code harder to read, so we can't merge this in. Take a look here: https://rules.sonarsource.com/javascript/RSPEC-3358.